partcad/src/partcad/healthcheck/stale_git_locks.py

```
@@ -0,0 +1,79 @@
+            return False
+        lock.release()
+        return True
+
+    def test(self) -> HealthCheckReport:
+        self._stale = []
+        try:
+            for lock_path in sorted(glob.glob(os.path.join(self._git_dir, "*.lock"))):
+                if self._is_unheld(lock_path):
+                    self._stale.append(lock_path)
+                    self.findings.append(f"Stale git lock file: {lock_path}")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)